packages/twenty-front/src/modules/ui/layout/show-page/components/ShowPageRightContainer.tsx

```diff
@@ -137,6 +140,13 @@ export const ShowPageRightContainer = ({
     isWorkflowEnabled &&
     targetableObject.targetObjectNameSingular ===
       CoreObjectNameSingular.WorkflowVersion;
+  const isWorkflowRun =
+    isWorkflowEnabled &&
+    targetableObject.targetObjectNameSingular ===
+      CoreObjectNameSingular.WorkflowRun;
+
+  const hideTabsNotWorkflowRelated =
```

Comment:
`isWorkflowRelated` would more consistent with the actual value

Reply:
Done